app/javascript/react/screens/App/Settings/screens/ConversionHostsSettings/components/ConversionHostWizard/ConversionHostWizardLocationStep/ConversionHostWizardLocationStep.js

```diff
@@ -1,10 +1,127 @@
 import React from 'react';
-import { Form } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import { reduxForm, Field } from 'redux-form';
+import { required } from 'redux-form-validators';
+import { Form, Spinner } from 'patternfly-react';
 
-const ConversionHostWizardLocationStep = () => (
-  <Form className="form-horizontal">
-    <h2>TODO: Location Step Contents</h2>
-  </Form>
-);
+import { BootstrapSelect } from '../../../../../../common/forms/BootstrapSelect';
+import {
+  V2V_TARGET_PROVIDERS,
+  RHV,
+  OPENSTACK,
+  FETCH_TARGET_COMPUTE_URLS
+} from '../../../../../../../../../common/constants';
 
-export default ConversionHostWizardLocationStep;
+import { stepIDs, PROVIDER_TYPES } from '../ConversionHostWizardConstants';
+
+class ConversionHostWizardLocationStep extends React.Component {
+  componentDidMount() {
+    const { fetchProvidersAction, fetchProvidersUrl } = this.props;
+    fetchProvidersAction(fetchProvidersUrl);
```

Comment:
Since we're fetching providers on mount, we could use this data to dynamically populate the provider type dropdown with only what is available.

Oh, that makes me think, what happens on the conversion hosts tab if there are no providers?  Do we show the provider empty state like in Mappings and Overview?

Reply:
That's a really good point, I hadn't considered that. I think as implemented, it would just show the conversion hosts empty state and prompt the user to open the conversion host wizard, where the providers dropdown would be empty. That's a case I need to handle.

Reply:
Also handled this.